pyo3-macros-backend/src/module.rs

```diff
@@ -1,26 +1,31 @@
 //! Code generation for the function that initializes a python module and adds classes and function.
 
-use crate::utils::Ctx;
 use crate::{
-    attributes::{self, take_attributes, take_pyo3_options, CrateAttribute, NameAttribute},
+    attributes::{
+        self, take_attributes, take_pyo3_options, CrateAttribute, ModuleAttribute, NameAttribute,
+    },
     get_doc,
+    pyclass::PyClassPyO3Option,
     pyfunction::{impl_wrap_pyfunction, PyFunctionOptions},
+    utils::Ctx,
 };
 use proc_macro2::TokenStream;
 use quote::quote;
 use syn::{
     ext::IdentExt,
     parse::{Parse, ParseStream},
     parse_quote, parse_quote_spanned,
+    punctuated::Punctuated,
     spanned::Spanned,
     token::Comma,
-    Item, Path, Result,
+    Item, Meta, Path, Result,
 };
 
 #[derive(Default)]
 pub struct PyModuleOptions {
     krate: Option<CrateAttribute>,
     name: Option<syn::Ident>,
+    module: Option<ModuleAttribute>,
```

Comment:
This new attribute probably needs documenting in the guide.

Reply:
Done. Sorry for the huge lag